Using-OnCloud.md

```diff
@@ -30,64 +56,44 @@ Set the `TARGET_SUBNET_ID` field to specify the subnet ID.
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: vpc-deployment-configmap
+  name: vpc-config
   labels:
-    hpcr: test
+    app: my-sample
 data:
   IBMCLOUD_REGION: us-east
   TARGET_SUBNET_ID: "xxx"
-  TARGET_CONTRACT_PUB_KEY_FILENAME: |
-    -----BEGIN PUBLIC KEY-----
-    MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAoCnfoQTXR+OJxEL1tYZs
-    +nm/y6f2Pr0Asgb4/8kmLgRchnbCUxsQQNhhgvk8iJZYKu+6CS0dKYId0X1Twm4W
-    H5NfOor4UYXHmTYHvqDmvCNKWByZk2xBAWEgPm76YtlQcsTJ01S0lNBVfIqs5gWN
-    o4Upv70uSPORyfINjZdMQ6a6mfI5Ittvbmx9c+VNKAXop3vVfUOlY1gFtKw9Kn/v
-    uKJZ/JJXzpLx72Gq1B5k1brfCINbhXDNB9KsU/zkry1Gk1sGwLTY0xb/BzYIyis3
-    +cPki+AyDmDOeBuVxYXC3j/ndWvlYiAIRVxn0zoJZxcsG9KqOoRaRRcNDYWEaNN9
-    mi7mOeBczkAveSr9Jxtun6tZ4PRK/eD1HFBAcu7PtK39OcLdsayrD8Cn+tDdIFqj
-    +lgHq4Z/rj11lRb9uk2aor0LnbbUhCeYQibrGN7hBz7wXm04MIpkUC1mNDhg2IuY
-    uaBImbT8vt8uqsvLeLWcQg87B+gMMcOiyRu9aKFuAXYV3xsu5OckvAL+S7x43Bis
-    TE2GSELABIxSgns5KniGu+V1EIN1AUJZMdECfgECuKmqCHvivy6IPa8I+y4QnXrj
-    SA5Ecni8I/tCAvmMCHAzFysbjLvPoAUSYiOlW969Kc5iJuBvq40WOR8xba3NwJji
-    CnzbhHxtOVAtLl5g92nLhv0CAwEAAQ==
-    -----END PUBLIC KEY-----
-  TARGET_PROFILE: bz2e-2x8  
 ```
 
 | ConfigMap Setting | Description |
 |-------------------|-------------|
 | IBMCLOUD_REGION   | The region where your VPC is located. |
-| TARGET_SUBNET_ID  | The subnet where you want to deploy the HPCR VSI. | 
-| TARGET_IMAGE_NAME | The **name** of the base image for the HPCR VSI. If this is not set, the controller will use the latest HPCR stock image. |
-| TARGET_PROFILE    | The **name** of the compute hardware profile to use to create the HPCR VSI. If this is not set, the controller defaults to `bz2e-2x8`. |
-| TARGET_CONTRACT_PUB_KEY_FILENAME | tbd |
-| IBMCLOUD_ZONE  (**Depreciated**)   | The zone within the VPC where you want to deploy the HPCR VSI. The controller only requires to specify the ID of the subnet, the VPC and the zone will be derived from the subnet.  |
-| TARGET_VPC_NAME  (**Depreciated**)   | The VPC where you want to deploy the HPCR VSI. The controller only requires to specify the ID of the subnet, the VPC and the zone will be derived from the subnet.  |
-
+| TARGET_SUBNET_ID  | The subnet where you want to deploy the HPCR VSI. |
```

Comment:
The `IBMCLOUD_REGION` is also important and it has to match the location of the subnet. 
It's a bit unfortunate that we have this redundancy, I'll investigate if it's possible to use the global search APIs to locate the subnet and then derive the region from the subnet.